src/Core/Checkout/Cart/Rule/LineItemCustomFieldRule.php

```diff
@@ -78,6 +78,12 @@ private function isCustomFieldValid(LineItem $lineItem): bool
     {
         try {
             $customFields = $lineItem->getPayloadValue('customFields');
+
+            if (empty($customFields)
```

Comment:
Maybe try to be more specific. Like $customFields !== null

Reply:
Wouldn't it be good to catch any other falsy value here?